tools/convert_cr_to_oc.py

```diff
@@ -21,21 +22,21 @@ def convertname(root, files, name):
 		os.unlink(os.path.join(root, "Names.txt"))
 		r = "\"$Name$\""
 	else:
-		print root, name
-	for lang, localname in names.iteritems():
+		print(root, name)
+	for lang, localname in names.items():
 		if "StringTbl" + lang + ".txt" in files:
-			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"rb").read().splitlines()
+			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"r").read().splitlines()
```

Comment:
I tested the changes on Windows 10 x64. It worked. The point is, with the binary mode, you would need to use bytes-objects instead of str-objects which sucks with different encodings.